pkg/core/writer/dimensions/client.go

```diff
@@ -175,7 +176,7 @@ func (dc *DimensionClient) processQueue() {
 			dc.Unlock()
 
 			if err := dc.setPropertiesOnDimension(delayedDim.Dimension); err != nil {
-				log.WithError(err).WithField("dim", delayedDim.Key()).Error("Could not send dimension update")
+				log.WithError(err).WithField("dim", delayedDim.Key()).Error("Could not Send dimension update")
```

Comment:
What's up with the random capitalization?

Reply:
Oh, looks like you did a s/send/Send/ and it did the string literals as well.

Reply:
Honestly, I have no idea what happened there?  Will fix it though.